llvm/lib/Transforms/IPO/FunctionAttrs.cpp

```diff
@@ -1864,6 +1864,23 @@ static bool InstrBreaksNonConvergent(Instruction &I,
          !SCCNodes.contains(CB->getCalledFunction());
 }
 
+static bool FunctionRequiresConvergence(const Function *F) {
```

Comment:
This is identical to `FunctionCalledWithConvergenceToken`. Can we move the definition to a common place, say `Function::hasConvergentCalls()`?